src/main/java/com/robinpowered/sdk/model/Identifier.java

```diff
@@ -72,6 +72,9 @@ public static Interface fromString(String value) {
     private final String value;
     private final DateTime createdAt;
 
+    // Mutable
+    private Integer deviceId;
```

Comment:
> Can an identifier ever not have a device?

Yes. In fact identifiers almost always have a short time where they're created but still not assigned to a device.

Even still, identifiers can be created for accounts/users that don't necessarily have to tie to a device.

A partial relic (but kinda nice ability) of our old buzzband days. 😆